firmware/config/engines/dodge_neon.cpp

```diff
@@ -502,11 +502,11 @@ void setDodgeNeonNGCEngineConfiguration(DECLARE_ENGINE_PARAMETER_SIGNATURE) {
 	engineConfiguration->tpsAccelLength = 12;
 	engineConfiguration->tpsAccelEnrichmentThreshold = 10;
 
-	engineConfiguration->suckedOffCoef = 0.05;
-	engineConfiguration->addedToWallCoef = 0.40;
+	engineConfiguration->wwaeTau = 1.0f;
```

Comment:
this is an approximate conversion of the old value, at sensible engine speeds.  It's also zeroed out 3 lines later, so I'm not worried.